playbooks/ps_80_kmip.yml

```diff
@@ -69,18 +69,37 @@
     with_items:
         - python3 -m pip install -U pip
         - python3 -m pip install -U setuptools
+        - python3 -m pip install urllib3==1.26.6  
         - python3 -m pip install -U pykmip
+    when: ansible_distribution_release != "bookworm"
+
+  - name: install pykmip
+    command: "{{ item }}"
+    with_items:
+        - python3 -m pip install -U --break-system-packages pip
+        - python3 -m pip install -U --break-system-packages setuptools
+        - python3 -m pip install -U --break-system-packages pykmip
+    when: ansible_distribution_release == "bookworm"  
 
   - name: downgrade SQLAlchemy as new version doesn't work with pykmip
     command: pip3 install SQLAlchemy==1.4.46
+    when: ansible_distribution_release != "bookworm"
+
+  - name: downgrade SQLAlchemy as new version doesn't work with pykmip
+    command: pip3 install --break-system-packages SQLAlchemy==1.4.46
+    when: ansible_distribution_release == "bookworm"  
 
   - name: start pykmip server
     shell: nohup /usr/local/bin/pykmip-server -f /package-testing/kmip/server.conf -l /pykmip_server.log 2>&1 &
-    when: ansible_distribution != "Amazon"
 
-  - name: start pykmip server for Amazon-Linux
-    shell: nohup /usr/bin/pykmip-server -f /package-testing/kmip/server.conf -l /pykmip_server.log 2>&1 &
-    when: ansible_distribution == "Amazon" and ansible_distribution_major_version == "2"
+  - name: Check if kmip-server is running
+    shell: ps -ef | grep kmip-server | grep -v grep
+    register: kmip_server_running
```

Comment:
Why not to fail the job here?

Reply:
If pykmip is not running - the DB won't start

Reply:
Done